src/components/Avatar/__stories__/avatar.stories.mdx

```diff
@@ -150,39 +150,6 @@ Use when a user’s image is not available, use their initials.
   </Story>
 </Canvas>
 
-### Initials avatar custom text format
```

Comment:
In the design review with Yael we decided to remove this story because we do not want people to override the appearance of the letters inside the avatar (it should be consistent)